runtime/runtime/request.go

```diff
@@ -66,8 +68,10 @@ type Request struct {
 	PathSegments httprouter.Params
 	MsgData      PubSubMsgData
 	Start        time.Time
-	Logger       zerolog.Logger
+	Logger       *zerolog.Logger
```

Comment:
Made this a pointer, as we had usecases where the zero value was being used by code to attempt to log - and it logged nowhere. Now the `Logger()` function can see if its' been set, and if not default to the RootLogger instead